src/krylov_phiv.jl

```diff
@@ -153,30 +148,31 @@ function ExponentialUtilities.expv!(w::GPUArraysCore.AbstractGPUVector{Tw},
         F = eigen!(SymTridiagonal(cache))
         expHe = F.vectors * (exp.(lmul!(t, F.values)) .* @view(F.vectors[1, :]))
     else
-        lmul!(t, cache)
-        expH = cache
-        _exp!(expH)
+        # lmul!(t, cache)
```

Comment:
keep this so it's non-allocating?

Reply:
The other `expv` functions calls `exponential!(t * cache, expmethod)`, so I mantained the same structure. Moreover, this function requires the `expmethod` argument, and `_exp!` does not have any `expmethod` I think.

Reply:
but why not `lmul!` the `t*cache` to remove the array allocation?